data/naltok outfits.txt

```diff
@@ -356,15 +356,15 @@ outfit "Acid Jet"
 	"illegal" 2500000
 	"unplunderable" 1
 	description "The Telashki apparently have no equivalent to the various treaties on warfare established by other species, and in few places is this clearer than in the Acid Jet. At the simplest level, the Acid Jet is essentially a pressure washer fed by large tanks of bacteria that produce an extremely powerful acid. The resulting weapon dissolves tissue, equipment and barriers with ease, making it very difficult to effectively defend against it."
-	description "This violates the vast majority of chemical and biological weapons legislation across the galaxy, and as such being caught with an Acid Jet outside of Naltok space is likely to result in a severe fine, if not a significant prison sentence."
+	description "	This violates the vast majority of chemical and biological weapons legislation across the galaxy, and as such being caught with an Acid Jet outside of Naltok space is likely to result in a severe fine, if not a significant prison sentence."
 
 outfit "Xochira Security Cracker"
 	category "Special"
 	cost 50000
 	thumbnail "outfit/naltok security cracker small"
-	"capture class" "xochira novelty"
-		"success" .6
-		"failure break" 1
+#	"capture class" "xochira novelty"
```

Comment:
I should probably just remove all the capture class stuff, seeing as the PR to add it isn't open any more, and the author has stated an intent to redesign the system.